index.js

```diff
@@ -1,4 +1,17 @@
 'use strict';
 const pReflect = require('p-reflect');
 
-module.exports = iterable => Promise.all(iterable.map(pReflect));
+module.exports = (iterable, opts) => {
+	const pLimit = require('p-limit');
```

Comment:
This needs to be top-level to be future-proof with eventual switch to `import` syntax.

Reply:
You could maybe use `mock.reRequire` in the tests.

Reply:
Done.